### DISCUSSION TITLE
Customise the model that generates the dialog title to prevent the title from being generated using an advanced model

### DISCUSSION THREAD

**Discussion by CodeZhangBorui** [open-webui/open-webui#5548] [General]
**Is your feature request related to a problem? Please describe.**

After I finished using the **o1-preview model** conversation, Open WebUI automatically generated the dialog title using the **o1-preview model**, which not only made the model think uselessly, but also wasted API billing.

**Describe the solution you'd like**

Use cheap, low-grade models to generate titles (e.g. 4o-mini).

Or, customise the model that generates the dialog title to prevent the title from being generated using an advanced model.

**Comment by cheahjs** [comment]
This is already configurable as the Task Model in Admin Settings.
